static/src/stylesheets/module/facia/_slices.scss

```diff
@@ -239,6 +210,29 @@ Hence why a greater depth of selector specificity is needed.
     }
 }
 
+.fc-slice__popular-mpu {
+    .ad-slot {
+        min-height: 274px;
```

Comment:
Well, it's all existing code, so it's okay if we don't have the chance to resolve it in this PR - but any cleanup is always an improvement!